lib/core/migrate/migrate.js

```diff
@@ -516,6 +516,13 @@ const migrations = [
 		async go() {
 			await Migrators.moveOption('keyboardNav', 'mediaBrowseMode', 'showImages', 'mediaBrowse');
 		},
+	}, {
+		versionNumber: '5.7.1-shard-newCommentCount',
+		async go() {
+			const tags = await Storage.get('RESmodules.newCommentCount.counts') || {};
+			const remappedTags = _.mapKeys(tags, (v, k) => `newCommentCount.${k.toLowerCase()}`);
+			await Storage.setMultiple(remappedTags);
```

Comment:
You can also `await Storage.delete('RESmodules.newCommentCount.counts');` here, since they're temporary anyways, doesn't really matter if the migration goes wrong